ts/Core/Chart/Chart.ts

```diff
@@ -477,6 +477,21 @@ class Chart {
             this.callback = callback;
             this.isResizing = 0;
 
+            if (optionsChart.zooming) {
+                optionsChart.zoomType = pick(
+                    optionsChart.zooming.type,
+                    optionsChart.zoomType
+                );
+                optionsChart.zoomBySingleTouch = pick(
+                    optionsChart.zooming.singleTouch,
+                    optionsChart.zoomBySingleTouch
+                );
+                optionsChart.pinchType = pick(
+                    optionsChart.zooming.pinchType,
+                    optionsChart.pinchType
+                );
+            }
+
```

Comment:
See if the filesize can be optimized by storing `optionsChart.zooming` in a local variable `zooming`. Another possible optimization to check would be `extend(optionsChart, { zoomType: pick(...`

Reply:
Thanks for the suggestions, adding a local variable reduced the file weight, extend increased.